internal/clustercontroller/cluster/rbac.go

```diff
@@ -16,6 +16,7 @@ func NewServiceAccountReconciler(
 	return reconciler.NewGenericResourceReconciler(&client, saName, saBuilder)
 }
 
+// enable this if cluster role is provided by adminitrator, and name must be "zookeeper-clusterrole"
 func NewClusterRoleBindingReconciler(
```

Comment:
**Suggestion:** The function `NewClusterRoleBindingReconciler` has a duplicated parameter `client`. Remove the duplicate to correct the function signature. [possible bug, importance: 9]